components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -1654,22 +1654,33 @@ def generateGuardedArray(self, array, name, specTemplate, specTerminator,
             if specTerminator:
                 currentSpecs.append(specTerminator)
             joinedCurrentSpecs = ',\n'.join(currentSpecs)
-            specs.append(f"&[\n{joinedCurrentSpecs}]\n")
+            specs.append(f"&Box::leak(vec![\n{joinedCurrentSpecs}].into_boxed_slice())[..]\n")
```

Comment:
I think this is the same as [Vec::leak](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.leak)